File: leetcode3/최원준/3894. Traffic Signal Color.py
#

'''
1. 아이디어 :
-

2. 시간복잡도 :
    O(1)

3. 자료구조/알고리즘 :
-

'''

class Solution:
    def trafficSignal(self, timer: int) -> str:
        return "Red" if 30<timer<=90 else "Orange" if timer == 30 else "Green" if timer == 0 else "Invalid"

File: leetcode3/이진희/3894_Traffic Signal Color.java
/*

1. 아이디어 : 시키는대로 조건문을 만들면 된다

2. 시간복잡도 : O(1)

3. 자료구조/알고리즘 : 단순 조건문

 */

class Solution {
    public String trafficSignal(int timer) {
        if(timer == 0) return "Green";
        else if(timer == 30) return "Orange";
        else if(timer > 30 && timer <= 90) return "Red";
        else return "Invalid";
    }
}